docs/video-miners/getting-started/testing/testnet.md

```diff
@@ -46,8 +48,11 @@ Once your orchestrator is running, check that you’re connected to the correct
 curl localhost:7935/EthChainID 
 ```
 
-If you are correctly connected to Rinkeby, this command will return `4`. 
+If you are correctly connected to Arbitrum Rinkeby, this command will return `421611`.
 
 **Please note that if you are running the Livepeer CLI with Docker, you will need to check port 7935 inside the go-livepeer container**.
 
-If your orchestrator is connected to Rinkeby, you are all set!
\ No newline at end of file
+
+7. **Register your service URI and fee structure on Arbitrum  using “Set orchestrator config”**
```

Comment:
Is this step really needed to get your orchestrator activated in Arbitrum? I remember playing around with Arbitrum and don't remember it was required.

Reply:
it's not strictly required to *activate* but it is required for the new O to be discoverable.

Reply:
this isn't super helpful for the Confluence testnet actions, but to interact with test broadcasters it will be important